.devops/code-review-pipelines.yml

```
@@ -13,7 +13,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local
-      ref: main
+      ref: CHK-4375-add-cdc-tests
```

```suggestion
      ref: main
```
TODO revert after https://github.com/pagopa/pagopa-ecommerce-local/pull/232 merge